src/app/shared/context.service.ts

```diff
@@ -198,7 +198,9 @@ export class ContextService implements Contexts {
               } as Notification);
               console.log(`Space with name ${val.space} and owner ${val.user}
                 from path ${val.url} was not found because of ${err}`);
-              return Observable.of({} as RawContext);
+                return Observable.throw(`Space with name ${val.space} and owner ${val.user}
```

Comment:
This needs to be un-indented, so that it is on the same level as the 'console.log'.